<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sumSys</class>
 <widget class="QDialog" name="sumSys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1291</width>
    <height>651</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1291</width>
    <height>651</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>统计系统</string>
  </property>
  <widget class="QWidget" name="left_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#DDDDDD</string>
   </property>
   <widget class="QListWidget" name="listFunc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget#listFunc{
    outline: 0px;
    border:0px;
    min-width: 120px;
    color: Black;
    background: #DDDDDD;
}

QListWidget#listFunc::Item{
     height:30px;
}
QListWidget#listFunc::Item:selected {
    background: rgb(49, 194, 124);
    border-radius:1.5px;
   
}
</string>
    </property>
    <item>
     <property name="text">
      <string>办卡续卡统计</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/card.svg</normaloff>resource/card.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>营业额统计</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/money.svg</normaloff>resource/money.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>办卡续卡流水</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/fluid.svg</normaloff>resource/fluid.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>入库出库统计</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/goods_price.svg</normaloff>resource/goods_price.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>入库出库流水</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/goods_fluid.svg</normaloff>resource/goods_fluid.svg</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>菜单栏</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>1061</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>办卡续卡统计</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gb_banka">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>391</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>某月的办卡续卡人数</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gb_bankhis">
     <property name="geometry">
      <rect>
       <x>439</x>
       <y>110</y>
       <width>601</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>近几个月的办卡续卡人数趋势</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>选择展示的年月</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>年：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_bar_year">
         <item>
          <property name="text">
           <string>2019</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2020</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2021</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>月：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_bar_month">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btn_show_bar">
         <property name="text">
          <string>展示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>选择显示多少个月</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>20</y>
        <width>571</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,0">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_show_num_month">
         <item>
          <property name="text">
           <string>3个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12个月</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_show_BXline">
         <property name="text">
          <string>展示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>营业额统计</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gb_money">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>831</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>点线图展示</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>选择显示多少个月</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>20</y>
        <width>571</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,0">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_show_num_month_money">
         <item>
          <property name="text">
           <string>3个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12个月</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_show_MoneyLine">
         <property name="text">
          <string>展示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>选择栏(默认可选近3年的流水)</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>411</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>年：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_fluid_year"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>月：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_fluid_month"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableView" name="tv_fluid">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>131</y>
       <width>961</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>操作栏</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>17</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_fluid_excrt">
         <property name="text">
          <string>导出当前流水</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_fluid_expall">
         <property name="text">
          <string>导出全部流水</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_fluid_getDB">
         <property name="text">
          <string>获取数据库流水信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>办卡续卡流水</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="gb_inout">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>130</y>
       <width>391</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>某月的入库出库数据</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>441</x>
       <y>40</y>
       <width>601</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>选择显示多少个月</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>30</y>
        <width>571</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9" columnstretch="1,0">
       <item row="0" column="0">
        <widget class="QComboBox" name="cb_show_num_month_2">
         <item>
          <property name="text">
           <string>3个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9个月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12个月</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_show_inoutLine">
         <property name="text">
          <string>展示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gb_inouthis">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>601</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>近几个月的入库出库以及利润统计</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>40</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>选择展示的年月</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0,1,0,0,1">
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>月：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_bar_inout_month">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_bar_inout_year">
         <item>
          <property name="text">
           <string>2019</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2020</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2021</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>年：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>货物：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6" rowspan="2">
        <widget class="QPushButton" name="btn_show_inout_bar">
         <property name="text">
          <string>展示</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="cb_inout_goods"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>入库出库统计</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>40</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>选择栏(默认可选近3年的流水)</string>
     </property>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>430</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1,0,1,0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>年：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>出/入库：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_fluid_year_2"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>月：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_fluid_month_2"/>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="cb_fluid_inout"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>40</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>操作栏</string>
     </property>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>17</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_fluid_excrt_2">
         <property name="text">
          <string>导出当前流水</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_fluid_expall_2">
         <property name="text">
          <string>导出全部流水</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_fluid_getDB_2">
         <property name="text">
          <string>获取数据库流水信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableView" name="tv_fluid_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>131</y>
       <width>961</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>入库出库流水</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_head">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>统 计 系 统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
